tests/test_pod.py

```diff
@@ -17,7 +17,7 @@
 def test_terminate_pods_by_name_pattern(cl, client, has_conf):
     has_conf.return_value = False
     pod = MagicMock()
-    pod.metadata.name = "my-app-1"
+    pod.metadata.name = "some-random-string-my-app-1"
```

Comment:
it is better if you can add these 3 tests, and maintain the existing one for backward compatibility.

Reply:
please add more tests, and maintain the existing ones.

Reply:
thanks, updated!